cypress/integration/login.js

```diff
@@ -20,6 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
-        cy.get('.calendar--selected').contains(15)
+        cy.get('.calendar--selected').contains(`${new Date().getDate()}`)
```

Comment:
'.calendar--selected' will fail if you'll click on a different calendar slot than today, you should construct a locator that will pin point exactly to today's slot.